depclean-gradle-plugin/src/main/java/se/kth/depclean/DepCleanGradleTask.java

```diff
@@ -0,0 +1,23 @@
+package se.kth.depclean;
+
+import org.gradle.api.Action;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.Project;
+import org.gradle.api.tasks.TaskAction;
+
+public class DepCleanGradleTask extends DefaultTask {
+
+  /**
+   * Action that analyzes the project dependencies and show the results.
+   */
+  @TaskAction
+  public void printDependencies() {
```

Comment:
The name of the function is not related to its body

Reply:
Can it be analyzeProject() or analyzeGradleProject()?